### DISCUSSION TITLE
docker adguard home cli update whitelist

### DISCUSSION THREAD

**Discussion by MeJIuFaRo** [AdguardTeam/AdGuardHome#6790] [Q&A]
Hello. Could you please advise me on whitelist-related question?

I have AdGuard Home installed in Docker. The whitelist file inside the container is located at /opt/adguardhome/work/data/filters/whitelist.txt

After adding a domain name to the file, I have to go to the web interface and manually click the "Check for updates" button. Even restarting the container does not update this list.

How i can update whitelist from cli in docker container ?

**Comment by ghost** [comment]
Try the below:
```
$ docker stop adguardhome
$ docker start adguardhome
```

**Reply by MeJIuFaRo** [reply]
ty for answer but it is don't work for me =(
